mlir/test/Dialect/Linalg/transform-op-match.mlir

```diff
@@ -39,6 +39,23 @@ transform.sequence failures(propagate) {
 
 // -----
 
+func.func @by_operand_type() {
```

Comment:
Would be nice to have a negative test case here, say to test the result if operand_type does not match